components/DropDownMenu/DropDownModal.js

```diff
@@ -232,16 +214,16 @@ class DropDownModal extends PureComponent {
       location: 0,
       color: changeColorAlpha(bufferColor, 1),
     }, {
-      location: Math.min(0.25, bufferHeight / screenHeight),
+      location: Math.max((bufferHeight - gradientHeight) / screenHeight, 0.15),
       color: backgroundColor
     }, {
-      location: (bufferHeight + gradientHeight) / screenHeight,
+      location: Math.max((bufferHeight / screenHeight), 0.2),
       color: invertedColor
     }, {
-      location: (bufferHeight + gradientHeight + transparencyHeight) / screenHeight,
-      color: invertedColor,
+      location: Math.min((bufferHeight + listViewHeight) / screenHeight, 0.8),
+      color: invertedColor
     }, {
-      location: Math.max((bufferHeight + listViewHeight) / screenHeight, 0.75),
+      location: Math.min((bufferHeight + listViewHeight + gradientHeight) / screenHeight, 0.85),
```

Comment:
Could we extract these decimal values into constants with descriptive names? Just so it's possible to understand what exactly these values are supp.osed to be and why they are such values. Ideally you could add a comment for each resulting `const`